examples/seismic/acoustic/acoustic_example.py

```diff
@@ -52,18 +52,18 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
 
     if preset == 'constant':
         # With  a new m as Constant
-        m0 = Constant(name="m", value=.25, dtype=np.float32)
-        solver.forward(save=save, m=m0)
+        v0 = Constant(name="v", value=2.0, dtype=np.float32)
+        solver.forward(save=save, vp=v0)
         # With a new m as a scalar value
```

Comment:
new vp*

Reply:
you forgot these comments @mloubout . Can fix in the next PR